indico/modules/rb_new/operations/blockings.py

```diff
@@ -110,3 +111,14 @@ def approve_or_request_blocking(blocking):
 
     for owner, rooms in rooms_by_owner.iteritems():
         notify_request(owner, blocking, rooms)
+
+
+def get_blocked_rooms(start_dt, end_dt, states=None):
+    query = (Room.query
+             .join(Room.blocked_rooms)
+             .join(BlockedRoom.blocking)
+             .filter(db_dates_overlap(Blocking, 'start_date', start_dt, 'end_date', end_dt)))
+
+    if states:
+        query = query.filter(BlockedRoom.state.in_(states or []))
```

Comment:
`or []` never happens

Reply:
ahh, yeah, forgot to remove it after refactoring